1_fetch/src/map_utils.R

```diff
@@ -0,0 +1,40 @@
+
+bbox_to_polygon <- function(bbox, bbox_crs = "+init=epsg:4326", return_crs = NULL) {
+  bbox_poly <- sf::st_sfc(sf::st_polygon(list(
+    matrix(bbox[c(1, 2, 1, 4, 3, 4, 3, 2, 1, 2)], ncol = 2, byrow = TRUE)),
+    dim = "XY"), crs = bbox_crs)
+  if (!is.null(return_crs)) {
+    sf::st_transform(bbox_poly, return_crs)
+  }
+  else {
+    bbox_poly
+  }
+}
+
+
+
+as_view_polygon <- function(object) {
```

Comment:
arg name could be a little more specific...maybe `view_config`?

Reply:
Since we are giving it an object that is a list that could have lots of different fields, I thought this pattern was closer to `viz` that other names that would be more specific.

Reply:
But the object must have fields for `bbox`, `projection`, and `aspect`, right? And we ignore all others? That makes it pretty much view-specific and bbox-specific IMO.